libr/include/r_anal.h

```diff
@@ -1702,6 +1703,8 @@ R_API char *r_anal_rtti_demangle_class_name(RAnal *anal, const char *name);
 R_API void r_anal_rtti_print_at_vtable(RAnal *anal, ut64 addr, int mode);
 R_API void r_anal_rtti_print_all(RAnal *anal, int mode);
 
+R_API void r_anal_colorize_bb(RAnal *anal, ut64 addr);
```

Comment:
add another argument to specify the color, or at least a number. like its done with the traces. See the dt command